### DISCUSSION TITLE
Building for multiple systems e.g. linux and windows

### DISCUSSION THREAD

**Discussion by idhamari** [nklayman/vue-cli-plugin-electron-builder#1806] [Q&A] [answered]
when running

                        yarn electron:build

It creates a desktop app for the current system. How can one build for other systems as well! it would be nice to add this [here](https://github.com/nklayman/vue-cli-plugin-electron-builder#to-build-your-app)

**Comment by klren0312** [comment]
you can see this
https://www.electron.build/multi-platform-build

**Comment by idhamari** [comment, marked as answer]
Thanks for your reply! 

In the link it says:

> Don’t expect that you can build app for all platforms on one platform.

So it seems this is not supported and one must have different systems to compile the app.
